package.json

```diff
@@ -123,7 +123,7 @@
     "jsonrepair": "^3.8.0",
     "keyboardjs": "^2.7.0",
     "lodash-es": "^4.17.21",
-    "lucide-react": "^0.503.0",
+    "lucide-react": "^0.539.0",
```

Comment:
The icon in Maja's design wasn't added until 0.508.0, but I bumped it to latest.